webapp/locales/en.json

```diff
@@ -15,7 +15,8 @@
     "followers": "Followers",
     "following": "Following",
     "shouted": "Shouted",
-    "commented": "Commented"
+    "commented": "Commented",
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-04-12T15:40:06Z" title="Friday, April 12th 2019, 5:40:06 pm +02:00">Apr 12, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -15,7 +15,8 @@
     "followers": "Followers",
     "following": "Following",
     "shouted": "Shouted",
-    "commented": "Commented"
+    "commented": "Commented",
```


Oh yes, providing translations right from the beginning :sparkling_heart:

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-04-12T16:12:21Z" title="Friday, April 12th 2019, 6:12:21 pm +02:00">Apr 12, 2019</time>_

---


:smile_cat: